api/src/main/java/module-info.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ * 
```

Comment:
Vlad, can you update to use a two date format here to stay consistent with our previous commits?

`Copyright (c) 2021-2021 Contributors to the Eclipse Foundation`